.cbmc-batch/stubs/aws_atomic_load_int.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Takes the existing list, and simply returns it. If the list originally nondeterminstic,
+ * this is sound (although it may lead to spurious proof failures if the code under test required the sorted property).
+ */
+
+#include <aws/common/atomics.h>
+
+/*
+ * For sequential proofs, we directly access the atomic value,
+ * so we can correctly propagate it through CBMC assumptions.
+ */
+size_t aws_atomic_load_int(const struct aws_atomic_var *var) {
+    return *((size_t *)(var->value));
+}
```

Comment:
@nchong-at-aws is it reasonable to have this stub for sequential proofs? It's not part of the code base and it is only invoked in certain proofs that use it in the context of assumptions.

Reply:
This is ok in a sequential setting used by CBMC since we do not check for data races.